### ISSUE TITLE
Failed to fetch

### ISSUE THREAD

**Issue by kerner1000** [atomiclabs/ledger-kmd-reward-claim#24]
1. Start Ledger device, oben the Komodo app.
2. Open https://atomiclabs.github.io/ledger-kmd-reward-claim/. (Tested with Firefox and Chrome)
3. Click "Check Rewards"
4 . Confirm on Ledger Live
5. Get "Failed to fetch" error.
6. Ledger Live asks for second confirm.

**Comment by palimarium** [comment]
Hi @kerner1000 , 

Ledger Nano S - Firmware 1.6.0
Komodo (KMD) Version 1.3.21
ledger-live-desktop-2.0.1-linux-x86_64.AppImage

[https://ibb.co/qFRRzrq](url)

I tried to check rewards after upgraded to the last version and I always get : "Failed to fetch" error.

I get the first confirmation message on my Ledger Nano S device for exporting the key and right after i click, I get the Failed to fetch error.
 
Thanks.

**Comment by mn851** [comment]
Yes, can confirm the KMD rewards claim tool no longer works with the latest version of Ledger Live (2.0.1) and recent Nano firmware update. Failed to fetch errors every time.

**Comment by brsh1** [comment]
> Yes, can confirm the KMD rewards claim tool no longer works with the latest version of Ledger Live (2.0.1) and recent Nano firmware update. Failed to fetch errors every time.

+1
same error here.
